src/test/cpp/gtester/ParsePopulationModel.cpp

```diff
@@ -10,7 +10,7 @@ namespace Tests {
 
 TEST(ParsePopulationModel, ParseDefaultPopulationModel)
 {
-	std::ifstream pop_file{"data/population_model_default.xml"};
+	std::ifstream pop_file{"../data/population_model_default.xml"};
```

Comment:
Ah, see, I was running the tests as `bin/gtester` from the `installed` directory… Relative paths are a real pain, here.

Do you think there’s maybe a way to construct a path relative from the *executable* directory, instead of the current working directory at the time the executable is launched? (I personally doubt it, and probably this isn’t worth much of our time; I’m fine running the tests from `installed/bin` instead.)